SConstruct

```diff
@@ -50,6 +45,8 @@ API_VERSION = f'{API_VERSION_MAJOR}.{API_VERSION_MINOR}.{API_VERSION_FIX}'
 
 def update_rpm_version(version, tag):
     """ Update the version (and release) in the RPM spec file """
+
+    # pylint: disable=consider-using-f-string
```

Comment:
Can we use f-strings in this file?

Reply:
Yes, but if you look at this file there's what should be a function stuck inside a giant `if` block, and it's for tagging releases on GitHub so it's not really tested.  What we should do is move it out of the top-level and put it in it's own code under site_scons somewhere.  Rather than rock the boat on that I just silenced the code-block.